applications/nrf5340_audio/src/utils/audio_nrf5340_dk_version.c

```diff
@@ -144,12 +144,12 @@ int board_version_get(struct board_version *board_rev)
 	return 0;
 }
 
-int board_version_valid_check(void)
+int audio_nrf5340_dk_version_valid_check(void)
```

Comment:
These and the ones above are for the audio dk, hence suggest 
```suggestion
int nrf5340_audio_dk_version_valid_check(void)
```

Reply:
See above.